run-ut.sh

```diff
@@ -207,6 +207,10 @@ fi
 
 export LD_LIBRARY_PATH=$STARROCKS_HOME/lib/hadoop/native:$LD_LIBRARY_PATH
 
+if [ "${USE_STAROS}" == "ON"  ]; then
+    export LD_LIBRARY_PATH=${STARROCKS_THIRDPARTY}/installed/starlet/starlet_install/lib64/:$LD_LIBRARY_PATH
```

Comment:
this will be a problem that breaks current be single binary principal, I will take a look how we can do it better. The good news is that be still has a script called `start_be.sh` as our last resort to deal with this LD_LIBRARY_PATH